dicom-archive/updateMRI_Upload.pl

```diff
@@ -3,6 +3,89 @@
 # zia.mohades@mcgill.ca
 # Perl tool to update the mri_upload table
 
+=pod
+
+=head1 NAME
+
+updateMRI_Upload.pl - updates database table C<mri_upload> according to an entry in table
+   C<tarchive>.
+
+=head1 SYNOPSIS
+
+updateMRI_Upload.pl [options] -profile prod -tarchivePath tarchivePath -source_location source_location
+
+=over 2
+
+=item *
+B<-profile prod> : (mandatory) path (absolute or relative to the current directory) of the 
+    profile file
+
+=item *
+B<-tarchivePath tarchivePath> : (mandatory) absolute path to the DICOM archive
+
+=item *
+B<-source_location source_location> : (mandatory) value to set column 
+    C<DecompressedLocation> to for the newly created record in table C<mri_upload> (see below)
+    
+=item *
+B<-globLocation> : Loosen the validity check of the DICOM archive allowing for the 
+     possibility that it was moved to a different directory.
+
```

Comment:
lower case for loosen to stay consistant with other options
Same for be verbose below

Reply:
Done